packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2358,6 +2358,21 @@ export class ContainerRuntime
 		this.dataStores.updateUnusedRoutes(dataStoreRoutes);
 	}
 
+	/**
+	 * This is called to delete objects from the runtime
+	 * @param unusedRoutes - object routes and sub routes that can be deleted
+	 * @returns - routes of objects deleted from the runtime
+	 */
+	public deleteUnusedNodes(unusedRoutes: string[]): string[] {
```

Comment:
naming: How about `deleteUnusedRoutes` to match the function above and the param name?

Reply:
Eventually my guess is that we'll remove `updateUnusedRoutes`. We are more deleting nodes than routes. There are potentially routes that exist that are not nodes.

Reply:
We're choosing not to change `updateUnusedRoutes` here, and instead making a separate function.